frontend/templates/troubleshooting/index.tsx

```diff
@@ -2,19 +2,73 @@ import { useRouter } from 'next/router';
 import { DefaultLayout } from '@layouts/default';
 import { DefaultLayoutProps } from '@layouts/default/server';
 import Head from 'next/head';
+import { IFixitAPIClient } from '@ifixit/ifixit-api-client';
+import React from 'react';
+import { useAppContext } from '@ifixit/app';
+import { Box, Container, Heading } from '@chakra-ui/react';
+
+type Problem = {
+   heading: string;
+   body: string;
+};
+
+type TroubleshootingData = {
+   title: string;
+   toc: string;
+   groups: Problem[];
+};
+
+const renderStyles = {
+   a: {
+      color: 'blue.500',
+   },
+   'ul, ol': {
+      paddingLeft: 4,
+   },
+};
 
 const Wiki: NextPageWithLayout<{ layoutProps: DefaultLayoutProps }> = () => {
+   const appContext = useAppContext();
+   const client = new IFixitAPIClient({ origin: appContext.ifixitOrigin });
+
    const { wikiname } = useRouter().query;
+   const [wikiData, setWikiData] = React.useState<TroubleshootingData | null>(
+      null
+   );
+   React.useEffect(() => {
+      client
+         .get(`Troubleshooting/${wikiname}`, {
+            credentials: 'include',
+         })
+         .then((res: any) => {
+            setWikiData(res);
+         });
+   }, [wikiname]);
```

Comment:
Shouldn't data be fetched server side (`getServerSideProps`) so that the page is server side rendered?

Reply:
Oh, good point. Done in 3d3ac9e0e126a7b9115eb97d57e7a2bf092c8386